pkg/scheduler/extender/extender.go

```diff
@@ -627,6 +658,40 @@ func createRequestFromPVCSpec(volumeName, storageType string, resourceRequiremen
 	}
 }
 
+func calculateScheduleTime(ctx context.Context, e *Extender, namespace string, podName string) (float64, float64, error) {
+	// record schedule information to metrics
+	cs, err := clientset.GetK8SClientset()
+	if err != nil {
+		e.logger.Errorf("cannot get clientset: %s", err)
```

Comment:
for such format %s,  better use err.Error(),  this is the string type.   err is not a string type.